x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/lib/mappings_validator.ts

```diff
@@ -235,6 +261,15 @@ export const validateMappingsConfiguration = (
     });
   }
 
+  if (Boolean(unknownConfigurationParameters.length)) {
```

Comment:
Minor nit: `unknownConfigurationParameters.length > 0` evaluates to a boolean too, and I think also more clearly expresses our intention here (similar comment for the condition on line 268).